packages/react-console/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@patternfly/react-console",
+  "version": "1.0.0",
+  "description": "This package provides VncConsole, SerialConsole and DesktopViewer React components to be used alongside patternfly-react to access virtual machine or server consoles.",
+  "main": "dist/js/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/js/index.d.ts",
+  "patternfly:src": "src/",
+  "sideEffects": [
+    "*.css"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "tag": "prerelease"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/patternfly/patternfly-react.git"
+  },
+  "keywords": [
+    "react",
+    "patternfly",
+    "console"
+  ],
+  "author": "Red Hat",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/patternfly/patternfly-react/issues"
+  },
+  "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/react-console#readme",
+  "scripts": {
+    "clean": "rimraf dist"
+  },
+  "dependencies": {
+    "@patternfly/patternfly": "4.59.1",
+    "@patternfly/react-core": "^4.75.9",
+    "@spice-project/spice-html5": "^0.2.1",
+    "@types/file-saver": "^2.0.1",
+    "@types/novnc-core": "^0.1.2",
+    "file-saver": "^1.3.8",
+    "lodash": "^4.17.19",
```

Comment:
Looks like react-console doesn't actually use lodash, could you remove this dependency?